rpc/json2/server.go

```diff
@@ -18,9 +18,10 @@ package json2
 
 import (
 	"encoding/json"
-	"github.com/ChainSafe/gossamer/rpc"
 	"log"
 	"net/http"
+
+	"github.com/ChainSafe/gossamer/rpc"
```

Comment:
Reason for this change? Shouldn't these be alphabetical?

Reply:
Simply ran the linting command with goimports enabled and it fixed accordingly....I have no preference

Reply:
changes are not mandatory nor related to this PR. Don't waste your time undoing them but please avoid changes like this in the future. Cheers to small and concise PRs!